src/memberdef.cpp

```diff
@@ -2113,7 +2113,7 @@ void MemberDefImpl::writeDeclaration(OutputList &ol,
     {
       doxyName.prepend(cdname+getLanguageSpecificSeparator(getLanguage()));
     }
-    ol.startDoxyAnchor(cfname,cname,anchor(),doxyName,doxyArgs);
+    if (inheritId.isEmpty()) ol.startDoxyAnchor(cfname,cname,anchor(),doxyName,doxyArgs);
```

Comment:
I think this condition should be part of `writeAnchor` otherwise doxygen will call `endDoxyAnchor` below at line 2435 without matching start.

Reply:
Indeed also the endDoxyAnchor should be protected (is only "used" with RTF and hence the problem didn't show up when checking HTML and LaTeX/PDF).